packages/admin-ui/src/filters/filter-base.tsx

```diff
@@ -3,7 +3,6 @@ import React, { useRef } from 'react'
 
 import { Button } from '../button'
 import { FilterPopoverFooter, FilterPopover } from './filter-popover'
-import { FilterDisclosure } from './filter-disclosure'
 
 import type { GenericFilterStateReturn } from './filter/filter.state'
 import { ComboboxList } from 'ariakit'
```

Comment:
seria melhor deixar o import mais específico para evitar possíveis problemas no tree shaking
```suggestion
import { ComboboxList } from 'ariakit/combobox'
```